coala_quickstart/generation/FileGlobs.py

```diff
@@ -42,3 +42,68 @@ def get_project_files(log_printer, printer, project_dir):
         ignored_file_paths=ignore_path_globs)
 
     return file_paths
+
+
+def generate_glob_exps(project_files, extensions, project_dir):
+    """
+    Generate glob expressions that matches exactly the files
+    with the given extensions in the user's project directory.
+
+    :param project_files:
+        The list of files to match.
+    :param extensions:
+        A list of file extensions.
+    :param project_dir:
+        The user's project directory.
+    :return:
+        A tuple with the first element (a list of glob expressions)
+        representing files to include and the second element (a list
+        of glob expressions) representing the files to exclude.
+    """
+    included_globs = []
+    excluded_globs = []
+
+    # Given a list of files, we condense the list into a list of glob
+    # expressions which match exactly those files. This will also include
+    # a list of exclude globs that exclude all matches.
+
+    # This is done by finding the list of all files in each directory and
+    # checking how many are present in the list of files we want to match.
```

Comment:
I feel like we're trying to be too smart, isn't it always the users glob adapted to the file ending? That's way simpler than creating a new glob and makes much more sense because we already have a glob.